app/services/sign_in/assertion_validator.rb

```diff
@@ -140,7 +148,8 @@ def expiration
     end
 
     def service_account_config
-      @service_account_config ||= ServiceAccountConfig.find_by(service_account_id:)
+      @service_account_config ||= ServiceAccountConfig.find_by(service_account_id: service_account_id ||
+                                                                                   nonvalidated_issuer)
```

Comment:
since this is only used once, I might just suggest we do `decoded_assertion_without_validation.iss` here, I think it's just confusing otherwise to have two different ways to get `issuer` throughout this file